lib/plain/common-types.ts

```diff
@@ -168,6 +176,21 @@ export type PlainClientAPI = {
     ): Promise<EnvironmentAliasProps>
     delete(params: OptionalDefaults<GetSpaceEnvAliasParams>): Promise<any>
   }
+  bulkAction: {
+    get<T extends BulkActionPayload = any>(params: GetBulkActionParams): Promise<BulkActionProps<T>>
+    publish(
+      params: GetSpaceEnvironmentParams,
+      payload: BulkActionPayload
```

Comment:
Do args need to use the specific types?

Reply:
Most of them define like such 🤔

Reply:
I'm saying that shouldn't this be BulkActionPublishPayload

Reply:
🤦🏽  yes! (fixed)